config-model/src/main/java/com/yahoo/vespa/model/admin/Admin.java

```diff
@@ -331,4 +325,25 @@ public boolean multitenant() {
 
     public ApplicationType getApplicationType() { return applicationType; }
 
+    public List<LogctlSpec> getLogctlSpecs() {
+        return logctlSpecs;
+    }
+    public void addLogctlCommand(String componentSpec, String levelsModSpec) {
+        logctlSpecs.add(new LogctlSpec(componentSpec,  levelsModSpec));
+    }
+
+    private static Set<LogctlSpec> setDefaultLogctlSpecs() {
```

Comment:
consider renaming this method to "alwaysLogctlSpecs()" or just "defaultLogctlSpecs()"

Reply:
Yup, that's better, fixed.